### ISSUE TITLE
Feature request: also find app in f-droid app store

### ISSUE THREAD

**Issue by k3b** [crisbal/PlayStoreLinks_Bot#14]
As a reddit user it would be nice to get infos from non-google-playstore, too

**Comment by crisbal** [comment]
Hey that's a good idea. 

I am not sure what is the best thing to do in term of user interaction for this, a separate command/trigger for the bot maybe?

Somehing like `linkmefdroid: ad away`.

**Comment by gondsm** [comment]
@crisbal why not show all stores by default in a clearly separated manner? That way, all stores would be featured equally!

**Comment by gondsm** [comment]
You are completely right, it could get confusing. The results from the various stores would have to be very clearly separated, perhaps using markdown headers, or something of that nature. But each entry could be presented as you've shown, just clearly identified as to which store it belongs!

**Comment by gondsm** [comment]
For instance (now I'm on a PC, I can reply properly) it could look something like

-----

### Play Store Result
[...]

-----

### F-Droid Result
[....]

-----

### Amazon Store Result
[...]

-----

Thus keeping stuff neat and clear for the user, in my opinion. @crisbal @k3b Thoughts?

**Comment by crisbal** [comment]
It is nice but I don't feel like it is the best choice. 

As of right now I already think there is too much clutter in the reply, adding more sections/headings would clutter the reply even more. The reason I created the bot was to aid mobile users, I don't feel comfortable in showing that many infos all the time. 

There is also to consider that most of the users don't really care for F-Droid or Amazon Store or anything else but Google Play (this is the first time I got this request in 4 years). I still think that having different triggers would be the best.

**Comment by gondsm** [comment]
Well, another option would be to make it into a table.

Anyway, different triggers would also work, of course. But in that case, could that option be displayed in the main reply? Just a single line saying something to the effect of "to display results from f-droid, reply with XXXX". What do you think?

**Comment by crisbal** [comment]
Yes, when I implement this I will of course advertise the feature as best as I can. I also have some other pending features I have yet to release (but are pushed here) so we could ship them together this one.

**Comment by gondsm** [comment]
Sounds like a plan! Many thanks and happy coding ;)
